src/rust/dbv2/migrations/2023-11-06-213153_price_levels/up.sql

```diff
@@ -0,0 +1,25 @@
+-- Your SQL goes here
+CREATE INDEX price_levels ON aggregator.user_history (order_status);
+
+
+CREATE VIEW api.price_levels AS
+    SELECT
+        market_id,
+        side,
+        price,
+        SUM(remaining_size) AS total_size
+    FROM
+        aggregator.user_history_limit
+    NATURAL JOIN
+        aggregator.user_history
+    WHERE
+        order_status = 'open'
+    GROUP BY
+        market_id,
+        side,
+        price;
```

Comment:
Is there any way we can sort by price ascending for asks and descending for bids?